docs/topics/sgx.md

```diff
@@ -291,21 +130,12 @@ spec:
     spec:
       containers:
       - name: sgxtest
-        image: oeciteam/sgx-test
-        command: ["/helloworld/host/helloworldhost", "/helloworld/enclave/helloworldenc.signed"]
-        volumeMounts:
-        - mountPath: /dev/sgx
-          name: dev-sgx
+        image: oeciteam/sgx-test:1.0
         securityContext:
```

Comment:
It doesnt need to run in privileged securityContext because of the Device Plugin.